apps/ledger-live-mobile/index.js

```diff
@@ -162,6 +165,21 @@ if (Config.DISABLE_YELLOW_BOX) {
 
 logReport.logReportInit();
 
+// Polyfill BigInt implementation
+const PolyfillBigInt = value => {
+  if (typeof value === "string") {
+    const match = value.match(/^0([xo])([0-9a-f]+)$/i);
+    if (match) {
+      return BigInteger(match[2], match[1].toLowerCase() === "x" ? 16 : 8);
+    }
+  }
+  return BigInteger(value);
+};
+
+if (typeof BigInt === "undefined") {
+  global.BigInt = PolyfillBigInt;
```

Comment:
BigInt will soon be available in React Native, but in the meantime I think it's very dangerous to do a global polyfill.

We tried in the past to add similar polyfills, and those have tendency and a great chance to create regression in our stack (e.g. on polkadot who feature detect the presence of BigInt, but may then uses more advanced feature). 

A recommended way may be to make the library that uses BigInt actually feature detect its availability. 🤔 

if we still consider this direction, the impact will be big in our QA and the risk to make regression is important.

Reply:
@gre all the CBOR libraries I have checked so far use BigInt internally. I am not sure how I can make them to detect BigInt internally without making it globally.

Reply:
@gre Is it enough if we polyfill the library that actually requires the BigInt implementation?

Reply:
I guess? I think the most important thing for us will be to identify the potential impact among all our dependencies on what it could impact, so we are sure to assess all the possible parts on whatever solution we go with :)

Reply:
I look for any reference to BigInt inside the ledger live project, and there is no one.

Reply:
you are right regarding "direct reference" to it, but indirectly we do have them in our dependencies.

```
grep -rnw . -e BigInt
```

(can also use `-l` for just the file list, lot of noises in that grep, some libs are dev libs too, but typically there is polkadot in the list)

Reply:
@gre do you think something like this would work? I am doing the same, but inside the package that needs to use BigInt... 
https://github.com/Zondax/node-cbor/blob/bfc1fc818550b9bfa3d6ae6604427c7bd1324f4a/packages/cbor/lib/cbor.js#L5-L18

Reply:
mmh I don't think so, because

```
  global.BigInt = PolyfillBigInt;
```

is global and can impact other libraries who uses `BigInt` from the (implicit) global context.

Reply:
@gre I have taken a different approach. I have updated the PR. I tested it locally with the emulator, and now it is working! Could you please check it?